### DISCUSSION TITLE
Use Apache as proxy server for budibase app, how to set the reverse proxy?

### DISCUSSION THREAD

**Discussion by cwmine** [Budibase/budibase#3911] [Help]
I use Apache as proxy server and set the budi base to https://xxx.xxx/budi like this:
<VirtualHost *:443>
   SSLProxyEngine On
	ProxyRequests Off
	ProxyPass /budi http://127.0.0.1:11090/
	ProxyPassReverse /budi http://127.0.0.1:11090/
        ProxyPreserveHost On
</VirtualHost>

However, when I open https://xxx.xxx/budi, it redirects to https://xxx.xxx/builder not https://xxx.xxx/budi/builder. The browser prompts 404 error

How to fix it？ 
Thanks.

**Comment by shogunpurple** [comment]
Hi @cwmine - budibase relies on the hostpaths so that the envoy proxy works - you can probably get this to work with some fairly in depth configuration on the apache side, but the easiest solution by far is to use a subdomain to host your budibase instance. At some point, we will enable support for host paths, maybe by being able to pass an override as an environment variable, but I can't give you a timeline for that, unfortunately.

Does that work for you?

**Reply by cwmine** [reply]
Could the bb url, change from http://127.0.0.1:11090/ http://127.0.0.1:11090/budi(or other phrase) as the root path in budi settings?
